docker/dev/docker-compose-portal-proxy.yml

```diff
@@ -45,7 +45,7 @@ services:
   app:
     environment:
       CONCORD_CONFIGURED_PORTALS: LOCALHOST HAS_STAGING
-      CONCORD_LOCALHOST_URL: ${PORTAL_PROTOCOL}://${PORTAL_HOST}/
+      CONCORD_LOCALHOST_URL: ${PORTAL_PROTOCOL:-http}://${PORTAL_HOST:?Need to define PORTAL_HOST}/
```

Comment:
I wasn't aware of this syntax, and even now, I am having trouble finding documentation for it.

I would expect to find something about this `:default` to be discussed in the docs, but cant find it here:
https://docs.docker.com/compose/compose-file/#variable-substitution

Or googling. 

Do you have a link to some documentation?  In particular I am wondering what the `-` or `?` mean.

Reply:
It isn't easy to find, but it is at the link you posted above, just scroll down a bit:
https://docs.docker.com/compose/compose-file/#variable-substitution:~:text=%24%7BVARIABLE%3A%2Ddefault%7D

(Good excuse for me to use chrome's special `:~:text=` argument. :) )

Reply:
Awesome -- and 🤯  `:~:text=` in URLs is amazing.